src/DataTable/TableCell.jsx

```diff
@@ -17,7 +22,10 @@ TableCell.propTypes = {
   render: PropTypes.func.isRequired,
   /** Table column */
   column: PropTypes.shape({
+    /** Class names for the cell wrapper */
     cellClassName: PropTypes.string,
+    /** Class names for the td element */
+    className: PropTypes.string,
```

Comment:
I'm not fully aware of the problem here, but it might be good to name this like `dataClassName` or `tdClassName` or something?

Or pulling back, this distinction seems so minute - block level vs inline? Is that a real distinction we want to expose? Could we move the `cellClassName` up to the `td` level safely, to solve the same problem you are looking at? Or is that an API break?

Reply:
I'm not sure what the original justification was for adding the `cellClassName` to the inline element vs the block element here, but I agree this feels a little redundant. Though, `.pgn__data-table-cell-wrap` has some styling that we may want to override (ex. `.pgn__data-table-cell-wrap` has a `max-width` and I found an edX repo that overrides it to none).

My use case for exposing the `td` element's className was to change its vertical alignment, which you can't access through the inline element.

I think given that there are use cases for both, it makes sense to expose both but update the names?

Reply:
You may need someone more in tune with paragon than me.

Is moving `.pgn__data-table-cell-wrap` up to `<td>` and dropping the `<span>` altogether a bad idea?

Reply:
I just tried to drop the span and update the styling and it didn't really go well, so I think that might be a bad idea